backend/src/index.ts

```diff
@@ -61,6 +61,15 @@ const session_remove = async (sessionId: string) => {
   return true;
 };
 
+const getUsername = async (userId: string) => {
+  const userData = query(users, where("__name__", "==", userId));
+  const user = await getDocs(userData);
+
+  if (user.empty) return 'user not found';
```

Comment:
i think returning `undefined` makes more sense here imo, means we can write something later (see: leaderboard/data) like `if (getUsername(etc)) { ... }` that throws an error if a username cant be found, cos that shouldnt happen give a proper userid